### ISSUE TITLE
 I hope you can provide a pre-trained model on UCF101.

### ISSUE THREAD

**Issue by xuejy19** [megvii-research/CVPR2023-DMVFN#4]
Hello, this is a great job! I am currently working on replicating this work. I have encountered a problem: when training the model on UCF101, I found that the model overfits after the 5th epoch. However, at this point, the performance of the model is far from the performance metrics given in the paper. So, I hope you can provide a pre-trained model on UCF101. Thank you!

**Comment by huxiaotaostasy** [comment]
Thank you for your question. There is indeed a phenomenon of overfitting, which you can alleviate by adjusting the learning rate and introducing stronger data augmentation. The pre-trained model in google drive named dmvfn_vimeo.pkl is trained on ucf, sorry for the misunderstanding.
